src/JuliusSweetland.OptiKey/UI/ViewModels/Management/DictionaryViewModel.cs

```diff
@@ -52,6 +53,32 @@ public string NewEntry
             }
         }
 
+        public bool DictionaryControlsAvailable
```

Comment:
I did it with the three properties so that if more suggestion methods are added it will be able to handle them. Seeing as that's not currently the case (and they can easily be re-added if more suggestion methods become available) I'll change it to use only one as you suggest.